packages/blade/src/components/Carousel/_decisions/decisions.md

```diff
@@ -21,16 +21,15 @@ Carousel is a UI component that allows the display and navigation of a set of co
 | navigationButtonPosition | `bottom,side`                          | `bottom`                                 | Sets the position of navigation button                                                                                                                                                                       |          |
-| navigationButtonSpacing  | `SpacingTokens`                        | `spacing.4`                              | Spacing between navigation button and slides when bleed is set to none                                                                                                                                       |          |
-| navigationButtonStyle    | `filled,stroke`                        | `filled`                                 | Styles to be used depending on the emphasis you want to give to the navigation buttons                                                                                                                       |          |
+| navigationButtonVariant    | `filled,stroke`                        | `filled`                                 | Styles to be used depending on the emphasis you want to give to the navigation buttons                                                                                                                       |          |
 | indicatorVariant         | `gray,white,blue`                      | `gray`                                   | Variants to be used depending on the background and the emphasis you want to give to the indicator.                                                                                                          |          |
 | onChange                 | `(slideIndex: number) => void`         | `undefined`                              | Callback which gets fired everytime a slide changes, useful for analytic events (only runs on user interaction not on autoPlay)                                                                              |          |
 | accessibilityLabel       | `string`                               | `undefined`                              | Accessibility label for the carousel, this will let screen reader users know what content the carousel holds (eg: "Product carousel")                                                                        |          |
 
-Additionally we will also expose few styled props, namely margin/width etc so that users can add negative margins to create end to end bleed carousel.
+- Additionally we will also expose few styled props, namely margin/width etc so that users can add negative margins to create end to end bleed carousel.
+- To let consumers scroll to any specific slide we will expose a `ref` with `goToSlide` method.
```

Comment:
Won't it be better to add controlled behavior instead 🤔 That can handle other cases as well where users might want more control

Reply:
Agree, will need to add controlled/uncontrolled states for that then. 

I think let's take it as an enhancement in upcoming sprints, need to fastrack the release for now.